.github/workflows/build-test-package.yml

```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.2rc02"
+            itk-git-tag: "c7586a12e8538c59041c480880ff0ea15c9550c2"
```

Comment:
The subject of the commit message says "Update builds for ITK 5.2 RC 3", so didn't you intend to specify `itk-git-tag: "v5.2rc03"`?

Reply:
Or possibly even `itk-git-tag: v5.2.0`?

Reply:
Windows CI has issues with the v5.2rc02 tag, so this is v5.2rc02++ :smile_cat: .

Reply:
@N-Dekker verifying that we have the opencl builds working with current elastix `develop`.